azure-pipelines.yml

```diff
@@ -1,4 +1,10 @@
 # Adapted from https://github.com/numba/numba/blob/master/azure-pipelines.yml
+trigger:
+- master
+
+pr:
+- master
```

Comment:
Do you know of a way to just do this for all PRs? I have a hunch that this wouldn't run now for backports but didn't see anything in the docs:

https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#pr-trigger

Reply:
Maybe just need to specify as *:

https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#wildcards

Though I think it's a bug that we need this in the first place given this in the docs:

> A pull request trigger specifies which branches cause a pull request build to run. If you specify no pull request trigger, pull requests to any branch trigger a build